apps/costcontrol/test/unit/settings_test.js

```diff
@@ -698,11 +698,13 @@ suite('Settings Test Suite >', function() {
         dataLimitInput.value = '124.56';
         dataLimitInput.dispatchEvent(evtInput);
 
-        var limitUnitValue = dataLimitSwitchUnitButton
-          .querySelector('span.tag');
-        assert.equal(limitUnitValue.textContent, 'MB');
+        assert.equal(dataLimitSwitchUnitButton.getAttribute('data-l10n-id'),
+          'unit-MB');
```

Comment:
nit: in Usage we usually align the function parameters like:
assert.equal(dataLimitSwitchUnitButton.getAttribute('data-l10n-id'),
                     'unit-MB');